modules/cnf-fast-event-notifications-api-refererence.adoc

```diff
@@ -35,6 +30,11 @@ The following API endpoints are available at [x-]`http://localhost:8089/api/clou
 * `/api/cloudNotifications/v1/health`
 - `GET`: Returns the health status of `cloudNotifications` API
 
+[NOTE]
+====
+8089 is the default port for the `cloud-event-proxy` container. You should use a different port for each of the `cloud-event-proxy` containers in the PTP Operator and application pods.
```

Comment:
PTP operator is always on 9085 and it cannot be changed.
This statement is valid only for DU apps